terminal/bundles/org.eclipse.terminal.view.ui/src/org/eclipse/terminal/view/ui/launcher/ITerminalConsoleViewManager.java

```diff
@@ -46,7 +46,7 @@ public interface ITerminalConsoleViewManager {
 	 *
 	 * @param tvid The terminals console view id.
 	 */
-	IViewPart showConsoleView(TerminalViewId tvid);
+	ITerminalsView showConsoleView(TerminalViewId tvid) throws CoreException;
```

Comment:
We should document that `CoreException` is thrown in case the console can not be opened.
Also `@return` would be nice, same applies below.

Reply:
done